flight/PiOS/STM32F4xx/pios_can.c

```diff
@@ -143,23 +144,36 @@ int32_t PIOS_CAN_Init(uintptr_t *can_id, const struct pios_can_cfg *cfg)
 	CAN_DeInit(can_dev->cfg->regs);
 	CAN_Init(can_dev->cfg->regs, (CAN_InitTypeDef *)&can_dev->cfg->init);
 
+	// Based on the selected RX IRQ we use either FIFO0 or FIFO1
+	if (can_dev->cfg->rx_irq.init.NVIC_IRQChannel == CAN1_RX1_IRQn ||
+		can_dev->cfg->rx_irq.init.NVIC_IRQChannel == CAN2_RX1_IRQn)
```

Comment:
Odd whitespace alignment here. At first glance, I thought there was an `if()` scoping failure. 

FWIW, this is why at work we violate the kernel style to always use braces (`{}`). We feel the odds of a bug sneaking in are too high and so for us it's not worth the tiny change in vertical spacing.